src/Components/calculatorUI.js

```diff
@@ -1,38 +1,33 @@
-import './calculatorUI.css';
-import React from 'react';
+import './CalculatorUI.css';
```

Comment:
- [ ] The app shows an error because the import at line 1. 'CalculatorUI.css' should not be capitalized.

Reply:
done